src/Generator/Commands/ActionGenerator.php

```diff
@@ -76,20 +76,99 @@ protected function getFilePath(): string
 
     protected function getStubFileName(): string
     {
-        return "actions/{$this->stub}.stub";
+        $file = new \Nette\PhpGenerator\PhpFile();
```

Comment:
I dont really like the style of this methods code much. it just looks quite messy, but i think we can clean it up quite a bit by segmenting it in separate methods and also clean up other stuff.

We probably want to have generic methods on the parent for things we do all the time and stuff like that. But lets see what we can do.

From the options you gave in the ticket, this is the best choice for a code generator package.

Reply:
Currently working on this ...